doc/source/whatsnew/v0.24.2.rst

```diff
@@ -66,7 +66,7 @@ Bug Fixes
 
 **Timezones**
 
--
+- Bug in :meth:`Timestamp.strptime` - support %z. (:issue:`21257`)
```

Comment:
can you move to 0.25.0

Reply:
also this is now an out-of-date message, put this in api-breaking changes, but change so it reflects the contents of the PR.